shared-clients/src/shared_clients/request_manager_client.py

```diff
@@ -210,11 +213,11 @@ async def send_message(
             "session_name": session_name or "",
         }
 
-        if debug:
-            print(
-                f"DEBUG: Sending request to {self.request_manager_url}/api/v1/requests/generic"
-            )
-            print(f"DEBUG: Payload: {message}")
```

Comment:
Great idea to use the logger! I don't know if the variable `debug` is still used somewhere or we can remove it.